README.md

```diff
@@ -28,16 +32,32 @@ jobs:
     name: Test the project
     steps:
     - uses: actions/checkout@v2
-    - uses: seisollc/goat@v0.3.0
+    - uses: seisollc/goat@v0.4.0
 ```
 
-and then store a custom dictionary file in `.github/etc/dictionary.txt` that accounts for your repo-specific language.  For example:
+#### Customizations
+1. Populate the custom dictionary file in `.github/etc/dictionary.txt` for any repo-specific language.
 ```bash
 $ cat << EOF >> .github/etc/dictionary.txt
 capricornis
 crispus
 EOF
 ```
+1. Configure the goat to skip terrascan scanning.
+```bash
+uses: seisollc/goat@v0.4.0
+with:
+  disable_terrascan: true
+```
+1. Provide a linting configuration for any of the linters `super-linter` uses.
+```bash
+$ mkdir -p .github/linters/
```

Comment:
Would expect 54 & 55 to throw the linter error as in 7 & 8 but they were not in the log.

Reply:
It only throws the error when all lines in a block begin with the `$`. Markdown style guidelines advise against using `$` unless you're using it to distinguish between entered commands and their output.

https://github.com/markdownlint/markdownlint/commit/0ec5e846be70641641aaf670c48dfe7972106e96
\<removed dead link\>